### PR TITLE
Add Popup menu

### PR DESCRIPTION
**Pull request by Katended**
The following have been addressed in in this pull request:

- Added background color
- Fixed background
- Add section anchors

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Katended and Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

<div align='center'>

</div>


You have submitted a project not having the given linters configuration for a html/css/js project. Kindly ensure that the `.github/workflows/linters.yml` file is [exactly as given](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: Index.html [hunk 1]
```diff
@@ -254,15 +261,19 @@ <h2>About Myself</h2>
...
       <h1 >Contact me</h1>
       <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
       <form action="https://formspree.io/f/mqkoywbg" method="POST" class="pure-form" name="">            
-          <input type="text" class="input-rounded" placeholder="Lorem Ipsum" maxlength="30" name="name" required>                          
-          <input  type="email" class="input-rounded" placeholder="loremipsum@gmail.com" required>             
+          <label for='Name' class="visuallyhidden">Name</label>
+          <input type="text" class="input-rounded" placeholder="Name" maxlength="30" name="name" required>                          
+          <label for='email' class="visuallyhidden">Email Address</label>
+          <input  type="email" class="input-rounded" placeholder="Email Address" required>             
+          <label for='message'class="visuallyhidden">Message</label>
```
Comment (zieeco) [inline Index.html:272]: - Hey :wave: , I see you've just added this <label> for Name, Email Address, and Message. You can add the labels but instead, the input should be nested in it. please, kindly remove the text content you've added to make your design consistent following the given Figma template. It should look like this. please, update this.
```
 <label for='email' class="visuallyhidden">
     <input  type="email" class="input-rounded" placeholder="Email Address" required>  
</label>
```

*Figma design*
Screenshot from 2023-04-18 16-28-15

*Your design*
Screenshot from 2023-04-18 16-27-04

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,





Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by PinkMoon25 — APPROVED** [review]
Hi @Katended , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linter checks :heavy_check_mark: 
Github flow :heavy_check_mark: 
Pair-programming :heavy_check_mark: 
Changes implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**